src/main/kotlin/com/ongdev/blog/api/models/dtos/responses/ChildCreationResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.ongdev.blog.api.models.dtos.responses
+
+import java.util.*
+
+class ChildCreationResponse (
```

Comment:
Sao lại cần cái này nhỉ, cái nào cũng là comment mà

Reply:
> Sao lại cần cái này nhỉ, cái nào cũng là comment mà

Em làm vậy để tách parent với children ra riêng ( nhưng vẫn giữ parent id của chúng), tùy theo gọi controller nào thì trả ra cái đó, em thấy có task là reply list for each comment